openmc/surface.py

```diff
@@ -27,6 +27,29 @@
 """
 
 
+class SurfaceCoefficient:
+    """Descriptor class for surface coefficients.
+
+    Parameters
+    -----------
+    symbol : str
+        Name of the coefficient
+
+    """
+    def __init__(self, symbol):
+        self.symbol = symbol
+
+    def __get__(self, instance, owner=None):
+        if instance is None:
+            return self
+        else:
+            return instance._coefficients[self.symbol]
+
+    def __set__(self, instance, value):
+        check_type('{} coefficient'.format(self.symbol), value, Real)
+        instance._coefficients[self.symbol] = value
+
+
```

Comment:
```suggestion
class ReadOnlySurfaceCoefficient:
    """Descriptor class for read only surface coefficients.

    Parameters
    -----------
    value : str or float
        Value of the read only coefficient. If the value is a string then the
        proper coefficient will be returned. If the value is a float it will
        be returned

    """
    def __init__(self, value):
        self.value = value

    def __get__(self, instance, owner=None):
        if instance is None:
            return self
        else:
            if isinstance(self.value, str):
                return instance._coefficients[self.value]
            else:
                return self.value

    def __set__(self, instance, value):
        raise AttributeError('This attribute is read-only')
        
        
```

Reply:
still trying to figure out how this reviewing doodad works... But let me know what you think about these changes